lib/topological_inventory/openshift/collector.rb

```diff
@@ -16,6 +17,7 @@ def initialize(source, openshift_host, openshift_port, openshift_token, default_
       self.collector_threads = Concurrent::Map.new
       self.finished          = Concurrent::AtomicBoolean.new(false)
       self.limits            = Hash.new(default_limit)
+      self.metrics           = TopologicalInventory::Openshift::Collector::Metrics.new
```

Comment:
I don't have a better name right now, but this might cause confusion if/when we start collecting actual openshift metrics as opposed to exposing metrics about the collector

Reply:
Yikes, yeah ... maybe `ApplicationMetrics` ? @Fryguy any opinion here? I'll probably make this change across all the services that use this pattern.